### DISCUSSION TITLE
Visualising colour pallet

### DISCUSSION THREAD

**Discussion by jameschetwood** [saadeghi/daisyui#2392] [Ideas / feature requests]
I've made a really simple React component showing the Daisy's color pallet. I load this in my site in dev mode as I find it useful to see the colours and make changes as I go. 

<img width="1430" alt="image" src="https://github.com/saadeghi/daisyui/assets/6142418/9d9aaaf2-75ec-4281-8248-b3eaa00a9bed">

Does anything like this exist already? If not would it be helpful for anyone if I published it as an NPM package?

**Comment by saadeghi** [comment]
There's a table [here](https://daisyui.com/docs/colors/#-2) where you can see all the colors for built-in themes.

But yes, it might be good idea to have a publicly available React component for that 👏

**Reply by jameschetwood** [reply]
I threw together a demo: https://daisyui-palette.vercel.app/ 

And here is the component which would be in the npm package: https://github.com/jameschetwood/daisyui-palette/blob/main/src/app/DaisyuiPalette.tsx 

Any other thoughts before I publish to npm? It's not mobile optimised but as I only use this when developing locally this hasn't bothered me.

**Reply by saadeghi** [reply]
>It's not mobile optimised

You can improve it later, not worries.

**Comment by jameschetwood** [comment]
Done

https://www.npmjs.com/package/daisyui-palette-demo
